importers/src/bible/osis/index.ts

```diff
@@ -328,14 +327,6 @@ export class OsisImporter extends BibleEngineImporter {
                 break;
             }
             case OsisXmlNodeName.LINE_GROUP: {
-                if (tag.isSelfClosing && tag.attributes.sID) {
```

Comment:
why remove these? I think this should only affect sword (the "sID" and "eID" I assume are only used by Sword, so shouldn't affect self-closing line groups that are non-Sword)

Reply:
self closing tags are part of the OSIS spec, including `sID` and `eID` - it wouldn't work without those attributes (this allows cross-overlapping structures in hierarchical XML (like paragraphs spanning chapters). i have several osis files that use them.

if you look at the beginning of `parseOpeningTag`, self closing end tags are already taken care of:
```
        if (tag.isSelfClosing) {
            context.openedSelfClosingTag = stackTag;
            // stop if this is actually a closing tag
            if (tag.attributes.eID) return;
        } 
```

there is equivalent code in `parseClosingTag`. So the code that i deleted is redundant in function and also makes the code hard to maintain/understand (closing a tag in `parseOpeningTag`) - as long as i'm not missing something here. 

concerning the removal of skipping the self-closing book tag in `parseOpeningTag`:  I haven't seen files that have tags like this, however it still looks like a bug, since if a file has a tag like this, it needs to be parsed.